addons/mail/static/tests/new/activity/activity_tests.js

```diff
@@ -41,6 +62,33 @@ QUnit.test("activity upload document is available", async function (assert) {
     assert.containsOnce(target, ".o-mail-activity .o_input_file");
 });
 
+QUnit.test("activity can upload a document", async function (assert) {
+    const pyEnv = await startServer();
+    const fakeId = pyEnv["res.partner"].create({});
+    const activityTypeId = pyEnv["mail.activity.type"].search([
+        ["name", "=", "Upload Document"],
+    ])[0];
```

Comment:
```js
    const [activityTypeId] = pyEnv["mail.activity.type"].search([
        ["name", "=", "Upload Document"],
    ]);
```

Reply:
Cleaner :) I updated the previous test since it use the same pattern